### PR TITLE
Newcomers_Guide.rst: Step 6 out of place bold text

### PR DESCRIPTION
**Pull request by sidharthskumar**
i removed the double quotes that enclosed -pull request from the rst
file which should solve the bold text text problem from index as well as
content region

Fixes https://github.com/coala/coala/issues/4442

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by JohnMoutafis** [comment]
ack e104b5d

**Comment by JohnMoutafis** [comment]
@sidharthskumar do a [`rebase`](http://api.coala.io/en/latest/Developers/Git_Basics.html#rebasing) just in case and you are ready to go I believe!

**Comment by sidharthskumar** [comment]
Sure ,thanks

**Comment by JohnMoutafis** [comment]
reack 82fcc26

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -318,7 +318,7 @@ where ``myfork`` is the name of your fork that you added at the previous step.
     You could also add a profile picture on your Github account, so that
     you can be distinguished out from the crowd!
 
-Step 6. Creating a ``Pull Request``
+Step 6. Creating a Pull Request
 -----------------------------------
```
Comment (jayvdb) [inline docs/Developers/Newcomers_Guide.rst]: Underline is too long?

Comment (sidharthskumar) [inline docs/Developers/Newcomers_Guide.rst]: oh am really sorry , it was actually my first PR. I ll surely be more careful next time

Comment (jayvdb) [inline docs/Developers/Newcomers_Guide.rst]: No problem

**Comment by sidharthskumar** [comment]
Oh OK.. am currently outside . LL do it when I return home

**Comment by sidharthskumar** [comment]
can someone delete this current  PR ? i have created a new one with all the changes and needs  a review at https://github.com/coala/coala/pull/4447 .

**Comment by IpshitaC** [comment]
@jayvdb Should a new PR be opened for this, or the changes be made to the existing PR?

**Pull request closed without merging**